apps/labs/posts/alt-text-scipy-2022.md

```diff
@@ -0,0 +1,72 @@
+---
+title: SciPy 2022 Accessibility Awareness Programs
+author: tony-fast
+published: July 12, 2022
+description: "Announcing the SciPy 2022 Accessibility Awareness Efforts"
+category: [Accessibility]
+# featuredImage:
+#   src: /posts/hello-world-post/featured.png
+#   alt: 'Excellent alt-text describing the featured image'
+hero:
+ imageSrc: /posts/ally-scavenger-hunt-post-hero-image.png
+ imageAlt: 'Three screenshots from the digital version of How to Write Alt Text for Scientific Diagrams and Visualization.'
+---
+
+It is SciPy this week! It's one of the biggest community events in our corner of the world. We are so excited for the festivities, the talks, and see hear all our friends new ideas.
+
+Events, and technology, are increasingly more complex these days. This year SciPy is in-person and virtual, and we wanted to bring y'all two things that fit these constraints:
+
+1. We're introducing a pilot open source [scavenger hunt](#the-scavenger-hunt) where you can:
+   1. Contribute to popular open source projects.
+   2. Improve the alt text of open source documentation.
+2. A free digital _and limited print_ zine called [__How to Write Alt Text for Scientific Diagrams and Visualization__][zine] by [@marsbarlee](https://github.com/marsbarlee).
+
+## The scavenger hunt
```

Comment:
If you're putting a period at the end of `## How to participate.`, you should also put one here.